backend/src/main/java/com/bounswe2024group10/Tradeverse/controller/AssetController.java

```diff
@@ -1,22 +1,39 @@
 package com.bounswe2024group10.Tradeverse.controller;
 
-import com.bounswe2024group10.Tradeverse.dto.asset.*;
-import com.bounswe2024group10.Tradeverse.service.AssetService;
+import java.util.List;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.bounswe2024group10.Tradeverse.dto.asset.AddAssetRequest;
```

Comment:
nit: Why are you importing all of the dto's one by one instead of using *;

Reply:
I put * but while saving the file editor changes it to multiple imports :/